packages/gestalt/src/ModuleExpandableItem.js

```diff
@@ -1,83 +1,88 @@
 // @flow strict
-import React, { Fragment, type Node } from 'react';
+import React, { type Node } from 'react';
 import Box from './Box.js';
+import Flex from './Flex.js';
 import Icon from './Icon.js';
+import ModuleTitle from './ModuleTitle.js';
 import TapArea from './TapArea.js';
 import Text from './Text.js';
-import { type BaseProps, type ExpandableItemProps } from './moduleTypes.js';
-import ModuleTitle from './ModuleTitle.js';
+import { type ModuleExpandableItemBaseProps } from './moduleTypes.js';
 
 type Props = {|
-  ...BaseProps,
-  ...ExpandableItemProps,
+  ...ModuleExpandableItemBaseProps,
+  accessibilityExpandLabel: string,
+  accessibilityCollapseLabel: string,
+  id: string,
+  isCollapsed: boolean,
+  onModuleClicked: (boolean) => void,
 |};
 
 export default function ModuleExpandableItem({
-  id,
-  title,
+  accessibilityCollapseLabel,
+  accessibilityExpandLabel,
+  children,
   icon,
   iconAccessibilityLabel,
-  accessibilityExpandLabel,
-  accessibilityCollapseLabel,
-  summary,
+  id,
   isCollapsed,
   onModuleClicked,
+  summary,
+  title,
   type = 'info',
-  children,
 }: Props): Node {
   return (
-    <Fragment>
```

Comment:
A component whose top-level export is a Fragment — aside from combos like Button + Dropdown — is a bit of a code smell. In this case, it meant that this component _had_ to be wrapped in a Box where used, which is leaky.

Reply:
Nice catch and good to know.